src/main/java/com/devonfw/ide/sonarqube/common/impl/DevonfwJavaProfile.java

```diff
@@ -0,0 +1,31 @@
+package com.devonfw.ide.sonarqube.common.impl;
+
+import org.sonar.api.server.profile.BuiltInQualityProfilesDefinition;
+import org.sonar.api.utils.AnnotationUtils;
+import org.sonar.plugins.java.api.JavaCheck;
+import org.sonarsource.api.sonarlint.SonarLintSide;
+
+/**
+ * @author lniazman
+ *
+ */
+@SonarLintSide
+public class DevonfwJavaProfile implements BuiltInQualityProfilesDefinition {
+
+  @Override
+  public void define(Context context) {
+
+    NewBuiltInQualityProfile devonfwJava = context.createBuiltInQualityProfile("Devonfw Java", "java");
+    org.sonar.check.Rule ruleAnnotation;
```

Comment:
please use `import org.sonar.check.Rule;` statement. FQN should only be used when required (if two types with the same simple name have to be used or to workaround classloading issues). Whenever I see an FQN I assume that it is some edge-case and otherwise I get confused.